query/query_test.go

```diff
@@ -168,11 +169,12 @@ func TestQuery(t *testing.T) {
 			parseInput: url.Values{
 				"search": []string{"foo", "bar"},
 				"age_eq": []string{"20"},
+				"year_eq": []string{"1998"},
 			},
 			expectedQueryInput: &DBQuery{
 				Limit:   25,
-				CondExp: "age = ? AND ((name = ? OR status LIKE ?) AND (name = ? OR status LIKE ?))",
-				CondVal: []interface{}{"foo", "%foo%", "bar", "%bar%", int64(20)},
+				CondExp: "age = ? AND year = ? AND ((name = ? OR status LIKE ?) AND (name = ? OR status LIKE ?))",
+				CondVal: []interface{}{"foo", "%foo%", "bar", "%bar%", int64(20), int(1998)},
```

Comment:
I think that there is no need to explicitly convert to `int` here, only `int64` is needed.

Reply:
Done